x-pack/plugins/observability/public/pages/slo_edit/slo_edit.tsx

```diff
@@ -33,7 +33,7 @@ export function SloEditPage() {
   const { sloId } = useParams<{ sloId: string | undefined }>();
   const { hasAtLeast } = useLicense();
   const hasRightLicense = hasAtLeast('platinum');
-  const { data: slo, isInitialLoading } = useFetchSloDetails({ sloId });
```

Comment:
@shahzad31 Why is this not needed anymore?

Reply:
this was un-necessarily blocking rendering of the form, causing empty page splash before form opens